README.md

```diff
@@ -42,8 +42,8 @@ Available only on GET step:
 
 Available on GET and EXPORT steps only:
 ```bash
---includeLanguages lang_1 [lang_2] [lang_3] ...
---excludeLanguages lang_1 [lang_2] [lang_3] ...
+--includeLanguages 'lang_1, lang_2, lang_3' ...
```

Comment:
Do not add spaces in examples, we prefer to not have them so better to educate users the way we would like to have things